ipt/src/tests.rs

```diff
@@ -0,0 +1,118 @@
+//! Unit tests for the IPT pallet.
+
+#![cfg(test)]
```

Comment:
You can remove these now, the whole file will skip compilation when tests arent running